composer.json

```diff
@@ -103,6 +103,7 @@
         "psr-4": {
             "Symfony\\Bridge\\Doctrine\\": "src/Symfony/Bridge/Doctrine/",
             "Symfony\\Bridge\\Monolog\\": "src/Symfony/Bridge/Monolog/",
+            "Symfony\\Bridge\\PhpUnit\\Compat\\": "src/Symfony/Bridge/PhpUnit/Compat/",
```

Comment:
Yeah, you're right. In fact, the PR build is green, great! https://travis-ci.org/symfony/symfony/builds/188846004